ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/deduplication/CompletionBasedDeduplicationPeriodConverter.scala

```diff
@@ -23,7 +23,7 @@ class CompletionBasedDeduplicationPeriodConverter(
       offset: Ref.HexString,
       applicationId: ApplicationId,
       actAs: Set[Ref.Party],
-      submittedAt: Instant,
+      recordTimeBound: Instant,
```

Comment:
Shouldn't be `maxRecordTime`?

Reply:
Probably would make more sense. Will change it in a follow up PR